test/info.py

```diff
@@ -231,6 +231,11 @@
     0x0311,  # K66F
     0x1101,  # Nordic-nRF52-DK
     0x1102,  # Nordic-nRF52840-DK
+    0x3104,  # dipdap_sdt52832b
+    0x3108,  # dipdap_sdt32429b
+    0x3110,  # dipdap_sdt32439b
+    0x3105,  # dipdap_sdt64b
+    0x3103,  # dipdap_sdt51822b
```

Comment:
Sorry… can you please keep the list sorted by ID? (This applies to any list in DAPLink and pyOCD that has IDs as the primary key.) Thanks!